torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -202,6 +204,19 @@ def _check_flat_params_on_expected_device(state: _FSDPState, module: nn.Module):
             )
 
 
+def _init_device_mesh(
+    pg: ProcessGroup,
+) -> Optional[DeviceMesh]:
+    # We are testing 1D DeviceMesh where dist.get_world_size(pg) == dist.get_world_size() for now.
+    # TODO: Address cases when dist.get_world_size(pg) != dist.get_world_size().
```

Comment:
There are 2 cases for the TODO, one is HSDP and another one is TP + FSDP. Can we add the cases to explain why `dist.get_world_size(pg) != dist.get_world_size()`?